meson.build

```diff
@@ -25,8 +25,8 @@ datadir = join_paths([prefix, get_option('datadir')])
 pkgdatadir = join_paths([prefix, get_option('datadir'), package_name])
 bindir = join_paths([prefix, get_option('bindir')])
 libexecdir = join_paths([prefix, get_option('libexecdir')])
-schemadir = join_paths([datadir, 'glib-2.0', 'schemas'])
```

Comment:
You don't need to care about making schemadir absolute. It is only used as the install_dir to a function. Meson functions for installing always accept relative paths (e.g. `share/glib-2.0/schemas`) and install the files relative to the prefix.

You do need to ensure this is an absolute path if you intend to do something else with schemadir, such as configure it into a file.

Reply:
Make sense, fixed. In this case I made a big mess as datadir already has prefix :fearful: 

At some point I'll rework the directory handling and pass relative paths and absolute when configuring.